kcidb/cache/__init__.py

```diff
@@ -157,3 +157,11 @@ def fetch(self, url):
         if blob.exists():
             return blob.download_as_bytes()
         return None
+
+    def empty(self):
+        """Empty the cache (remove all contents)."""
+        bucket = self.client.bucket(self.bucket_name)
+        blobs = bucket.list_blobs()
+
+        for blob in blobs:
```

Comment:
Looks good. Probably not efficient for large buckets, but should do alright for our usecase. Could you merge this and the previous line, though? I.e. do not assign the result of `list_blobs()` to a variable and use it directly in the loop? It's kinda trivial, and we could save some code lines this way.

Reply:
Done, Have a look now?